wordpress-mobile/WordPress-iOS
# Notifications Crash Failsafe

### Details:
I've not been able to repro this crash. This one _looks like_ an SDK / Swift glitch, nevertheless, i'd like to split the guard, just in case it's not being evaluated as expected.

Closes #5979

### To test:
1. Launch WPiOS.
2. Open the Notifications Tab.
3. Open any notification, and verify the details show up.

Needs review: @diegoreymendez 
Diego, may i bug you with a quick check?
